exampleconfig.cfg

```diff
@@ -0,0 +1,8 @@
+# these lines are ignored
```

Comment:
A fair point. It also occurs to me that I should update the readme to reflect the new format